beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/handler/UnreferencedMessageHandler.java

```diff
@@ -30,64 +33,86 @@
 import com.expedia.apiary.extensions.receiver.common.event.DropTableEvent;
 import com.expedia.apiary.extensions.receiver.common.event.ListenerEvent;
 
+import com.expediagroup.beekeeper.core.model.EntityHousekeepingPath;
 import com.expediagroup.beekeeper.core.model.LifecycleEventType;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.EventTypeListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.ListenerEventFilter;
-import com.expediagroup.beekeeper.scheduler.apiary.filter.MetadataOnlyListenerEventFilter;
+import com.expediagroup.beekeeper.scheduler.apiary.filter.LocationOnlyUpdateListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.TableParameterListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.WhitelistedListenerEventFilter;
-import com.expediagroup.beekeeper.scheduler.apiary.model.EventModel;
+import com.expediagroup.beekeeper.scheduler.apiary.model.UnreferencedEventModel;
 
 @Component
-public class UnreferencedMessageHandler extends MessageEventHandler {
+public class UnreferencedMessageHandler extends MessageEventHandler<EntityHousekeepingPath, UnreferencedEventModel> {
 
   private static final Logger log = LoggerFactory.getLogger(UnreferencedMessageHandler.class);
+  private static final String UNREFERENCED_DATA_RETENTION_PERIOD_PROPERTY_KEY = "beekeeper.unreferenced.data.retention.period";
```

Comment:
Where did this annotation used to live?

Reply:
This used to be in the config yaml file. Since its not supposed to be configurable, I moved it. Is that ok?